app/controllers/api/v1/books_controller.rb

```diff
@@ -18,6 +18,11 @@ def index
       def show
         render(json: Book.find(params['id']), serializer: ShowBookSerializer)
       end
+
+      # Summary: Get a book info by isbn
+      def book_info
+        render(json: OpenLibraryService.new.book_info(params['isbn']))
```

Comment:
if the service returns nil maybe it's better to show some kind of cool message instead of just a nil json :D